### DISCUSSION TITLE
FR: Filter/sort products based on if they are on sale (shopify's compare at price)

### DISCUSSION THREAD

**Discussion by austriker27** [statamic-rad-pack/shopify#128] [Feature Requests]
I would love to see the ability to filter (and sort) products that are on sale. (As mentioned in #47) 

Ideally the filter would look like this I think:

```
 {{ collection:products limit="12"  filter="saleIsCurrent" }}

            {{ partial:components/product_card }}

{{ /collection:products }}
```

Background: 
I currently have a [sale page on my website](https://rainierwatch.com/shop/sale/) and am hoping to replicate it.

**Comment by ryanmitchell** [comment]
Would a query scope not let you do this? https://statamic.dev/extending/query-scopes-and-filters#scopes

**Comment by jackabox** [comment]
@ryanmitchell yes it would, but would be nice to make a default way also.
